server/ai_live_video.go

```diff
@@ -234,7 +234,12 @@ func startTrickleSubscribe(ctx context.Context, url *url.URL, params aiRequestPa
 			seq := trickle.GetSeq(segment)
 			clog.V(8).Infof(ctx, "trickle subscribe read data received seq=%d", seq)
 
-			n, err := copySegment(segment, multiWriter)
```

Comment:
Are we still using the `copySegment` ? If not, it can be removed

Reply:
We still use it if there is no timeout set